pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -237,10 +264,19 @@ func TestLocalTaskRef(t *testing.T) {
 				Tektonclient: tektonclient,
 			}
 
+			if tc.localTaskRefResolverWithoutNamespace {
+				lc.Namespace = ""
+			}
```

Comment:
we could consider adding a `namespace` field into  `testcases` and set it to lc?

Reply:
+1